libs/constants.js

```diff
@@ -1,6 +1,12 @@
+import getConfig from 'next/config'
+
+
+const { serverRuntimeConfig, publicRuntimeConfig } = getConfig()
+
+
```

Comment:
For this and line 3, we have ESLint configured to error for two consecutive blank lines.